be/src/exprs/agg/window.h

```diff
@@ -463,7 +415,8 @@ class LeadLagWindowFunction final : public ValueWindowFunction<PT, LeadLagState<
         this->data(state).is_null = false;
         const Column* data_column = ColumnHelper::get_data_column(columns[0]);
         const InputColumnType* column = down_cast<const InputColumnType*>(data_column);
-        this->data(state).value = column->get_data()[frame_end - 1];
+        AggDataTypeTraits<PT>::assign_value(this->data(state).value,
+                                            AggDataTypeTraits<PT>::get_row_ref(*column, frame_end - 1));
```

Comment:
Since the specialization `LeadLagWindowFunction<PT, Slice, StringPTGuard<PT>>` is removed, will `BinaryColumn` also use this `assign_value`? which is like `value=ref`, where `value` and `ref` are both `Slice`.

Reply:
For `TYPE_BINARY`:
- The `ValueType` is `Buffer<uint8_t>`
- The implementation of `assign_value` is `memcpy(value.data(), ref.data, ref.size)`, at [assign_value](https://github.com/mofeiatwork/starrocks/blob/826548657472a7c0340503f82245f26eae867901/be/src/exprs/agg/aggregate_traits.h#L66)